README.md

```diff
@@ -38,12 +38,14 @@ module "grant_cloudtruth_access" {
 | role\_name | The role within your AWS account that cloudtruth will assume to perform its actions | `any` | n/a | yes |
 | s3\_policy | A custom policy to use for s3 instead of the one this module would define | `string` | `""` | no |
 | s3\_resources | The s3 resources to explicitly grant access to, defaults to all, and listing<br>all buckets is always allowed (for bucket chooser in UI) even if access<br>isn't granted here | `list(string)` | <pre>[<br>  "*"<br>]</pre> | no |
-| services\_enabled | The AWS services to grant cloudtruth access to, allowed values are s3, ssm, secretsmanager | `list(string)` | n/a | yes |
-| services\_write\_enabled | The AWS services to grant cloudtruth write access to, allowed values are s3, ssm, secretsmanager | `list(string)` | `[]` | no |
+| services\_enabled | The AWS services to grant cloudtruth access to, allowed values are s3, ssm, secretsmanager, kms | `list(string)` | n/a | yes |
+| services\_write\_enabled | The AWS services to grant cloudtruth write access to, allowed values are s3, ssm, secretsmanager, kms | `list(string)` | `[]` | no |
```

Comment:
We're not actually granting write access to kms, just using kms to encrypt stuff we write to the other services, should probably rword this

Reply:
Good catch.  I'll come up with clearer docs.